projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -150,15 +150,10 @@ control_service_publish_job_base_image-secure:
     - export VERSION_TAG="1.$CI_PIPELINE_ID"
     - bash -ex ./publish-job-base.sh
   retry: !reference [.control_service_retry, retry_options]
-  only:
```

Comment:
@tozka  I don't think we need the except clause here because it is not in the folder where the changes must happen. 
Do you agree?

Reply:
Do you mean this except clause 
```
except:
    changes:
      - projects/control-service/projects/helm_charts/pipelines-control-service/version.txt
```
Then yes, we do not need it.

Yes, we do not need it. We had it at times when Control Service releases were manual. Since we did not want to trigger everything just to make new release . We had "release" pipelines that only ran the release job. 

As releases of Control Service are automated , this except is no needed at all.

Reply:
But I'd prefer if you remove them in separate PR if possible/feasible.

Reply:
DOne